js_1_4.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+
+    'use strict'
+
+
+    //タスク一覧を作成
+    const taskList = [];
+
+    //一覧を表示
+    const taskShow = () => {
+      console.log('==================');
+      console.log('現在持っているタスク一覧');
+      console.log('==================')
+      
+      taskList.forEach((task,index) => {
+        const taskgenre = [];
+        Object.keys(task).forEach(key => {
+          taskgenre.push(task[key]);
+        });
+        const [content,genre] = taskgenre;
+        console.log(`${index}:[内容]${content},[ジャンル]${genre}`);
+      })
+
+    }
+    // タスク追加処理
+    const taskAdd = (genre,content) => {
+      const newTask = {
+        genre:genre,
+        content:content,
+      }
+      taskList.push(newTask);
+    }
```

Comment:
以下のように少し省略しても良いかなと思いました。
```suggestion
    const taskAdd = (genre, content) => {
        taskList.push({
            genre: genre,
            content: content,
        });
    }
```